### PR TITLE
Feature/pelagos 3796 add reports

### PR DESCRIPTION
**Pull request by praneethpr**
Need to merge PELAGOS-3792, and PELAGOS-3793 before merging this PR

### CODE REVIEW

**Comment by mickel1138** [comment]
I'm not getting any data on the download report?

**Comment by praneethpr** [comment]
> I'm not getting any data on the download report?

Yes, even i'm not getting any data on that report. Looking into it

File: src/Controller/UI/ReportResearchGroupDatasetStatusController.php [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+    // Limit the research group name to this to keep filename length at 100.
+    const MAXRESEARCHGROUPLENGTH = 46;
+
+    /**
+     * The default action.
+     *
+     * @param Request $request Message information for this Request.
+     * @param EntityHandler $entityHandler
+     * @param integer $researchGroupId The identifier for the Research Group subject of the report.
```
Comment (fightingtexasaggie) [inline src/Controller/UI/ReportResearchGroupDatasetStatusController.php]: GCS alignment

File: src/Controller/UI/ReportResearchGroupDatasetStatusController.php [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+     *
+     * @return Response|StreamedResponse A Symfony Response instance.
+     * @throws \Exception
```
Comment (fightingtexasaggie) [inline src/Controller/UI/ReportResearchGroupDatasetStatusController.php]: What throws this exception?

Comment (praneethpr) [reply src/Controller/UI/ReportResearchGroupDatasetStatusController.php]: I think PhpStorm added it when imported the entityHandler class, will remove it

**Pull request merged**